src/app/utils/JournalUtils.js

```diff
@@ -98,49 +114,90 @@ export function shipFromLoadoutJSON(json) {
         if (module.Engineering) _addModifications(ship.bulkheads.m, module.Engineering.Modifiers, module.Engineering.Quality, module.Engineering.BlueprintName, module.Engineering.Level, module.Engineering.ExperimentalEffect);
         break;
       case 'powerplant':
-        const powerplant = _moduleFromFdName(module.Item);
+        let powerplant = _moduleFromFdName(module.Item);
```

Comment:
Aha. Apparently I was wrong about the `_...` convention 😂  Okay...

Reply:
:)